src/EnsembleKalmanProcess.jl

```diff
@@ -189,23 +196,32 @@ function EnsembleKalmanProcess(
     # timestep store
     Δt = FT[]
 
+    # set accelerator ## will change to allow for additional accelerators in future
+    if isnothing(accelerator)
+        acc = DefaultAccelerator()
+    else
+        acc = NesterovAccelerator(accelerator.r, params)
```

Comment:
This is kind of awkward, since you are recreating the `Accelerator` struct. Can you use the `update_state` function here instead?